x/wasm/simulation/operations.go

```diff
@@ -53,14 +54,19 @@ func WeightedOperations(
 	)
 	simstate.AppParams.GetOrGenerate(simstate.Cdc, OpReflectContractPath, &wasmContractPath, nil,
 		func(_ *rand.Rand) {
-			// simulations are run from the `app` folder
-			wasmContractPath = "../x/wasm/keeper/testdata/reflect.wasm"
+			wasmContractPath = ""
 		},
 	)
 
-	wasmBz, err := os.ReadFile(wasmContractPath)
-	if err != nil {
-		panic(err)
+	var wasmBz []byte
+	if wasmContractPath == "" {
+		wasmBz = testdata.ReflectContractWasm()
+	} else {
+		var err error
+		wasmBz, err = os.ReadFile(wasmContractPath)
+		if err != nil {
+			panic(err)
```

Comment:
How about adding the unittest of this lines?

Reply:
This is dead code, so I deleted it.